### PR TITLE
Update readme: add build status badge and update text

### PR DESCRIPTION
**Pull request by gavanderhoorn**
As per subject.

Mentions the `motoman_experimental` repository, and provides some minor rationale for branch names.

6c6c38b (text update) is optional. I can remove it if desired.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
+
+Older releases may be found in the old ROS-Industrial [subversion repository][].
 
-[ROS-Industrial]: http://www.ros.org/wiki/Industrial
-[ROS wiki]: http://ros.org/wiki/motoman
-[SVN repo]: https://code.google.com/p/swri-ros-pkg/source/browse
+[ROS-Industrial]: http://wiki.ros.org/Industrial
+[ROS wiki]: http://wiki.ros.org/motoman
+[motoman_experimental]: https://github.com/ros-industrial/motoman_experimental
+[subversion repository]: https://code.google.com/p/swri-ros-pkg/source/browse
```
Comment (gavanderhoorn) [inline README.md]: Should this perhaps point to [ros-industrial/swri-ros-pkg](https://github.com/ros-industrial/swri-ros-pkg)?

Comment (shaun-edwards) [inline README.md]: +1 for updating the link.  The google code site is an archive and not easy to navigate.

Comment (gavanderhoorn) [inline README.md]: Done.

**Comment by gavanderhoorn** [comment]
@shaun-edwards: fixed the link.

**Comment by shaun-edwards** [comment]
+1...thanks for the contribution.

**Pull request merged**